installer/docker/ubuntu/jdk/Dockerfile

```diff
@@ -0,0 +1,55 @@
+# (C) Copyright IBM Corporation 2017.
```

Comment:
We haven't really discussed explicit Copyright notices in source files.  I'm ok with leaving this in place, but it then encourages all to add in their notices for each copyrightable change.  Note FWIW that there is a NOTICE file in the repo that indicates parts are (c) IBM already.